internal/metarepos/storage.go

```diff
@@ -420,24 +420,24 @@ func (ms *MetadataStorage) UnregisterStorageNode(snID types.StorageNodeID, nodeI
 
 func (ms *MetadataStorage) registerLogStream(ls *varlogpb.LogStreamDescriptor) error {
 	if len(ls.Replicas) == 0 {
-		return verrors.ErrInvalidArgument
+		return status.Errorf(codes.InvalidArgument, "empty replicas")
 	}
 
 	topic := ms.lookupTopic(ls.TopicID)
 	if topic == nil {
-		return verrors.ErrInvalidArgument
+		return status.Errorf(codes.InvalidArgument, "empty replicas")
```

Comment:
It seems that the topic does not exist. How about the NotFound error code in this situation?